resources/MedPromptX-VQA.md

```diff
@@ -0,0 +1,71 @@
+# MedPromptX-VQA
+
+<div align="center">
+    <a href="https://github.com/openmedlab/"><img width="700px" height="auto" src="https://i.ibb.co/dB9cYxS/Med-Prompt-X.jpg"></a>
+</div>
+<p style="text-align:center;font-size:10px;"><em></em></p>
+
+## Dataset Information
+
+MedPromptX-VQA is derived from a unified multimodal dataset, denoted as HAIM-MIMIC-MM. HAIM-MIMIC-MM is a fusion of information sourced from MIMIC-IV and MIMIC-CXR-JPG databases, meticulously curated to focus solely on patients with at least one chest X-ray procedure. HAIM-MIMIC-MM encapsulates records from 7,279 hospitalization stays, involving 6,485 distinct patients, thereby establishing a multimodal link encompassing tabular, textual and visual representations of patient health data.
+
+## Dataset Meta Information
+
+| Task Type | Language | Test                            | File Format | Size |
+|-----------|----------|---------------------------------|-------------|------|
+| VQA       | English  | 968 Image Question-Answer Pairs | .csv, .jpg  | 1.8GB |
+
+
+## Dataset Information Statistics
+
+In MedPromptX-VQA, patients are labeled with 12 pathological conditions: _Atelectasis, Cardiomegaly, Consolidation, Edema, Enlarged Cardiomediastinum, Fracture, Lung Lesion, Lung Opacity, Pleural Effusion, Pleural Other, Pneumonia and Pneumothorax_. To alleviate the challenges of limited context length and hallucination in LLMs, we transformed these labels into a binary single-label classification framework to ensure that the input fits the context length and to acquire a controlled output, rather than acquiring an open set of possible diagnoses. For each label, if a patient exhibits the condition, the corresponding label is assigned the value 1; otherwise, it is given the value 0. For this study, we specifically selected patients diagnosed with the aforementioned conditions, resulting in 968 records split into 501 positive and 467 negative samples.
```

Comment:
```suggestion
In MedPromptX-VQA, patients are labeled with 12 pathological conditions: _Atelectasis, Cardiomegaly, Consolidation, Edema, Enlarged Cardiomediastinum, Fracture, Lung Lesion, Lung Opacity, Pleural Effusion, Pleural Other, Pneumonia and Pneumothorax_. To alleviate the challenges of limited context length and hallucination in LLMs, we transformed these labels into a binary single-label classification framework. This ensures that the input fits the context length and provides a controlled output, rather than an open set of possible diagnoses. For each label, if a patient exhibits the condition, the corresponding label is assigned the value 1; otherwise, it is given the value 0. For this study, we specifically selected patients diagnosed with the aforementioned conditions, resulting in 968 records split into 501 positive and 467 negative samples.
```